packages/mermaid/src/diagrams/flowchart/flowDb.js

```diff
@@ -763,6 +764,11 @@ const makeUniq = (sg, allSubgraphs) => {
   return { nodes: res };
 };
 
+const replaceUnicodeTitleWithVal = (title) => {
+  const normalized = title.replace('u:u-', '').replaceAll('-', ' ').toUpperCase();
+  return getSymbolByTitle(normalized);
+};
+
```

Comment:
This will add unicode support only to flowcharts, we should be doing it across all diagrams like we do for comments and frontmatter. 

https://github.com/mermaid-js/mermaid/blob/24c9506935d353eaea8f234c9f8ab910b284bbfe/packages/mermaid/src/Diagram.ts#L48-L50

Reply:
@sidharthv96 probably the message above was lost because of the resolved conversation, could you please check it?

Reply:
Hmmm, yeah that's a problem. Maybe we replace after the rendering is done?
Maybe after this? 
https://github.com/mermaid-js/mermaid/blob/0ef115f69ab7c0e1f0d7ca880991f914fe9e437f/packages/mermaid/src/mermaidAPI.ts#L529-L530

Reply:
https://github.com/mermaid-js/mermaid/blob/0ef115f69ab7c0e1f0d7ca880991f914fe9e437f/packages/mermaid/src/diagrams/common/common.ts#L45-L57

This function is called from most places that handle text nodes. 
Doing Unicode replacement inside the `sanitizeText` function is a bit sketchy, but it's better than having partial support. 

You can try this if replacing `svgCode` messes up the diagram spacing. Test is with some long-named-unicode symbols.

Reply:
I moved it to `cleanUpSvgCode`, not an ideal place but it seems to be doing kind of a similar job 6b4f9fe069c9d03a28da242992ddd8f3c31a3830
also, it renders correctly with the very long title 4a4ecf54d478b89e369e21e9ce307ee968e1c574